python_bindings/bindings.cpp

```diff
@@ -146,6 +146,7 @@ template<typename dist_t, typename data_t = float>
 class Index {
  public:
     static const int ser_version = 1;  // serialization version
+    static const int file_handle_count = 4; // number of file handles to open for persistent index
```

Comment:
Is this just so we can read it from the python bindings? How would we use this / how would a user use this?

Reply:
Makes me think we ought to refactor `setupPersistentIndexFileHandles` to be a const array of const structs with members and locations we can pass to a single method, but that's probably overkill for this PR.

Reply:
Yeah its just so the python code is not hardcoding the file count.

I agree with that, I also think the multi-index management ought to be shoved into c++.

Reply:
Why does python need to know about the file count?

Reply:
In https://github.com/chroma-core/chroma/pull/891 we allow python to manage multiple index file handle status in order to buffer against OS limits.